Zend/tests/object_types/missing_type_hint_inheritance_in_interface.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+Missing interface method a object type hint during inheritance
+--FILE--
+<?php
+
+interface One {
+    public function a(object $obj);
+}
+
+interface Two extends One {
+    public function a($obj);
```

Comment:
This seems to be normal PHP behaviour, but contravariant subclasa parameters doesn't really break LSP. Can't really be changed in this patch though